.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.11.4
+      - image: circleci/node:10.24.1
```

Comment:
some of the newer packages aren't compatible with node 8, so we'll keep it close and go to 10.